### PR TITLE
Remove redundant try/catch

### PR DESCRIPTION
**Pull request by marimeireles**
Does it look good?
Thanks @martinRenou !

### CODE REVIEW

File: src/xeus_sqlite_interpreter.cpp [hunk 1]
```diff
@@ -115,20 +115,17 @@ void interpreter::load_db(const std::vector<std::string> tokenized_code)
 
 void interpreter::create_db(const std::vector<std::string> tokenized_code)
 {
-    try
+    if (std::ofstream(tokenized_code[2].c_str()))
```
Comment (martinRenou) [inline src/xeus_sqlite_interpreter.cpp:118]: Mmh I am not sure this is ok. You might need to close this `ofstream`. What about keeping the same code? So just
```cpp
m_bd_is_loaded = true;
m_db_path = tokenized_code[2];

// Create the file
std::ofstream(m_db_path.c_str()).close();

// Create the database
m_db = std::make_unique<SQLite::Database>(m_db_path, SQLite::OPEN_READWRITE | SQLite::OPEN_CREATE);
```

Comment (marimeireles) [reply src/xeus_sqlite_interpreter.cpp:118]: I think `ofstream` is RAII and I need a condition for the `if`.

Comment (martinRenou) [reply src/xeus_sqlite_interpreter.cpp:118]: Even if it's RAII, you are reading from the file before the object is deleted, so I do think you need to `close` it before reading again from it with the sqlite API.

Why do you think you need an `if` condition?

Comment (marimeireles) [reply src/xeus_sqlite_interpreter.cpp:118]: You mean that I should leave the error to SQLite to throw?

Comment (marimeireles) [reply src/xeus_sqlite_interpreter.cpp:118]: I can also close it later inside the if body.

Comment (martinRenou) [reply src/xeus_sqlite_interpreter.cpp:118]: > You mean that I should leave the error to SQLite to throw?

Well that what you were doing before, no? You were catching a `std::runtime_error`, when does this `std::runtime_error` happen?

Comment (marimeireles) [reply src/xeus_sqlite_interpreter.cpp]: That's true. Yeah, I'll change it.

**Pull request merged**